astropy/cosmology/connect.py

```diff
@@ -237,8 +322,56 @@ class CosmologyToFormat(io_registry.UnifiedReadWrite):
         Keyword arguments passed through to data writer.
     """
 
-    def __init__(self, instance, cls):
+    def __init__(self, instance: Cosmology, cls: type[Cosmology]) -> None:
         super().__init__(instance, cls, "write", registry=convert_registry)
 
-    def __call__(self, format, *args, **kwargs):
+    # ===============================================================
+    # __call__ overloads
+
+    # To a Cosmology
```

Comment:
Such trivial comments are not helpful, the return type annotation should be clear enough by itself. 
```suggestion
```

Reply:
SGTM

Reply:
Done!